chain/actions/Cargo.toml

```diff
@@ -58,5 +58,6 @@ anyhow = { workspace = true }
 mockall = { workspace = true }
 hex-literal = { workspace = true }
 tokio = { workspace = true }
+test-log = { workspace = true }
```

Comment:
Not `tracing_test`? We could unify those 2.

Reply:
did not use that yet